iOSTest/View Controllers/LoginViewController.swift

```diff
@@ -28,19 +28,75 @@ class LoginViewController: UIViewController {
     // MARK: - Properties
     @IBOutlet weak var usernameTextfield: UITextField!
     @IBOutlet weak var passwordTextfield: UITextField!
+    
+    @IBOutlet weak var usernameView: UIView!
+    @IBOutlet weak var passwordView: UIView!
+    
     private var client = LoginClient()
+    var username:String?
+    var password:String?
     
     // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
         
         title = "Login"
+        roundViewCorners()
     }
     
     // MARK: - Actions
     
     @IBAction func didPressLoginButton(_ sender: Any) {
         
+        let startTime = CACurrentMediaTime()
+        
+        username = usernameTextfield.text
+        password = passwordTextfield.text
```

Comment:
You can move these lines of code to ```validateUsernameAndPassword()```. It's confusing to see this here because you only begin to use ```username``` & ```password``` in line 79 in ```validateUsernameAndPassword()```.